source/common/types/cardanoNode.types.js

```diff
@@ -24,9 +42,64 @@ export const CardanoNodeStates: {
   ERRORED: 'errored',
 };
 
-export type TlsConfig = {
-  port: number,
-  ca: Uint8Array,
-  cert: Uint8Array,
-  key: Uint8Array,
+export type CardanoPidOptions = (
+  'mainnet-PREVIOUS-CARDANO-PID' |
+  'staging-PREVIOUS-CARDANO-PID' |
+  'testnet-PREVIOUS-CARDANO-PID' |
+  'development-PREVIOUS-CARDANO-PID' |
+  string
+);
+
+export type DaedalusPidOptions = (
+  'mainnet-PREVIOUS-DAEDALUS-PID' |
+  'staging-PREVIOUS-DAEDALUS-PID' |
+  'testnet-PREVIOUS-DAEDALUS-PID' |
+  'development-PREVIOUS-DAEDALUS-PID' |
+  string
+);
+
+export type CardanoNodeStorageKeys = {
+  PREVIOUS_CARDANO_PID: CardanoPidOptions,
+  PREVIOUS_DAEDALUS_PID: DaedalusPidOptions
+};
+
+// TODO: Add more options when the exact process names are established
```

Comment:
@cleverca22 we need your help with this part.
If I am not mistaking we have:
- `Electron` as process name in development AND in the Linux production, and
- `Daedalus` as process name in production for OSX and Win
Is that right?

Reply:
@cleverca22 don't waste time on this one - we will remove this part of the code...